examples/go/tagging/gotag-fn/handler.go

```diff
@@ -0,0 +1,177 @@
+package function
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+
+	handler "github.com/openfaas-incubator/go-function-sdk"
+	"github.com/pelletier/go-toml"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/vapi/rest"
+	"github.com/vmware/govmomi/vapi/tags"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+// tagInfo is part of the toml vcconfig file
+type tagInfo struct {
+	Action string
+	Urn    string
+}
+
+// VCenter is part of the toml vcconfig file
+type vCenterAuth struct {
+	Password string
+	Server   string
+	User     string
+}
+
+// Config represents the toml vcconfig file
+type vcConfig struct {
+	Tag     tagInfo
+	VCenter vCenterAuth
+}
+
+// The request from vmware-event-router.
+type dataObj struct {
```

Comment:
These types are really hard to read. You might want to copy the event payload structure from the vmware-event-router events package.

Reply:
I agree--I really have to get up to date on how to name things. I'll take another look at your event router. I know it probably seems obvious to you that I should look at things like vmware-event-router, but in my mind, the connection wasn't made and I don't know we're naming the same things. LOL So, thanks for mentioning it :)

Reply:
I made the changes as close to the vmware-event-router events package. What do you think?

Reply:
Thanks very much for your suggestion. I am now using it:
```
type incoming struct {
	Data types.Event `json:"data,omitempty"`
}
```